src/Std/Sync/StreamMap.lean

```diff
@@ -0,0 +1,147 @@
+/-
+Copyright (c) 2025 Lean FRO, LLC. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Sofia Rodrigues
+-/
+module
+
+prelude
+public import Std.Data
+public import Init.System.Promise
+public import Init.Data.Queue
+public import Std.Internal.Async.IO
+public import Std.Internal.Async.Select
+public import Std.Internal.Async.Basic
+
+public section
+
+open Std.Internal.Async.IO
+open Std.Internal.IO.Async
+
+/-!
+This module provides `StreamMap`, a container that maps string keys to async streams.
```

Comment:
I think you forgot to adjust this docstring, and also the docstring for `StreamMap`.